### ISSUE TITLE
HOLY SHIT BRO WTF

### ISSUE THREAD

**Issue by HausemasterIssue** [yoink-inc/konas#2]
HOWWW???? LITERALLY BEST DEOBF EVER THANK YOU

**Comment by moaji** [comment]
> HOWWW???? LITERALLY BEST DEOBF EVER THANK YOU

This is full source. Not a de obf.

**Comment by HausemasterIssue** [comment]
holy fuck you guys are a legend

**Comment by moaji** [comment]
> holy fuck you guys are a legend

I didn't do anything. It's just obvious since the comments are all there and the fact that package and method names are fine.

**Comment by HausemasterIssue** [comment]
@moaji two questions

1. is it a rat?
2. is it buildable, like i can use gradle and build it then slap it in my mods folder?

**Comment by moaji** [comment]
> @moaji two questions
> 
>     1. is it a rat?
> 
>     2. is it buildable, like i can use gradle and build it then slap it in my mods folder?
1. I'm not sure. Look for yourself. Check the gradle wrapper size aswell. If you are worried if it's a rat. Simple solution is to not use it.

2. I haven't tried building it since I don't have a use for the horrible client. From other issue posts it seems they built it.

**Comment by Gopro336** [comment]
> @moaji two questions
> 
> 1. is it a rat?
> 2. is it buildable, like i can use gradle and build it then slap it in my mods folder?

It might be a rat, I haven't checked it yet, given that this is yoink, I wouldn't trust it yet

yes, its buildable, get ready for all the "Konas +" skids lol.

**Comment by storm772** [comment]
well check for every file if you think it's a rat, its literally ***OPEN SOURCE***

**Comment by dopepushing** [comment]
> > @moaji two questions
> > 
> > 1. is it a rat?
> > 2. is it buildable, like i can use gradle and build it then slap it in my mods folder?
> 
> It might be a rat, I haven't checked it yet, given that this is yoink, I wouldn't trust it yet
> 
> yes, its buildable, get ready for all the "Konas +" skids lol.

nigga yoink is not even around anymore he doesnt do shit

**Comment by dopepushing** [comment]
Yeah,,,, Dis is a Remote Access Trojan. Build it and ur Ratted.

**Comment by HausemasterIssue** [comment]
thats what they said about poopboob and itwisten, until they realized nocom was a thing 💀

**Comment by OlDestiny** [comment]
tristen never quit and popbob has nothing to do with nocom @HausemasterIssue

**Comment by HausemasterIssue** [comment]
yeah tristan plays const now, but popbob still in nerds inc, so he had to
at least heard of it.

**Comment by OlDestiny** [comment]
he _used_ to be in nerds inc but left

**Comment by HausemasterIssue** [comment]
so where's he know? quit? i don't believe it.

**Comment by OlDestiny** [comment]
im pretty sure if he was still playing he would change his skin lmao

**Comment by HausemasterIssue** [comment]
he went by kenzie, i saw a player on const by the name of KenzieSchaffer,
that's 99% not popbob, and the last time the popbob ign logged onto const
was 9 months ago so it's not like dead dead.

**Comment by 6sn** [comment]
go outside

**Comment by HausemasterIssue** [comment]
This issue is like 3 months old lol. No I'm not a fitfag, and the account KenzieSchafer is Fornicator.
